composables/__tests__/useCustomerCreditCards.test.ts

```diff
@@ -0,0 +1,126 @@
+import { updatePaymentCard, deletePaymentCard, addPaymentCard } from "@/lib/gql/mutations"
+import { getPaymentCards } from "@/lib/gql/queries/checkout/getPaymentCards"
```

Comment:
Should be imported from @/lib/gql/queries. Add in index file. Follow the mutations import above.